csharp/ql/integration-tests/all-platforms/cshtml_standalone_net6/cshtml.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
Should this say `net6.0`?

Reply:
Thanks. Yes that makes sense. The .net 6 sdk has no knowledge of `net8.0` at all, so it has an impact on the success of the initial `dotnet restore`.

I pushed another change too to merge some asp.net core detection checks.